coordinator/src/services/validation/parameters-validator.ts

```diff
@@ -25,18 +24,12 @@ export function verifyParameters(
 
   const incorrectValueErrors = []
 
-  const ownerParameter = getOwnerParameter(parameters)
-  if (ownerParameter) {
-    const bodyOrg = getIdentifierValueForSystem(
-      ownerParameter.resource.identifier,
-      "https://fhir.nhs.uk/Id/ods-organization-code",
-      `Organization.identifier`
+  const resourceParameters = parameters.parameter.filter(isResourceParameter)
```

Comment:
why have we made this change? cant we stick with `getOwnerParameter`?

Reply:
Yesterday you mentioned writing some code here instead of just having the "getOwnerParameter" line by itself, so that it was clear what that line was doing when it was throwing an error for the test, is this not what you meant?

Reply:
Because the error is thrown inside of "getOwnerParameter"